### DISCUSSION TITLE
Adding a field on the goalpage (Help needed)

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1667] [Q&A] [answered]
@MoBosse @brockfanning @LucyGwilliamAdmin 

In my migration report I have data from different years. I want to add a field on the golapage which shows the current year of the data, similar like the status complete etc.

![image](https://user-images.githubusercontent.com/94443931/173054638-db514658-24df-4b1c-998a-80754e24d6fd.png)

What I did:
1: I copied the tags.html and pasted it into my _includes/components/indicator folder: https://github.com/christoph-LSN/MT_Site/blob/develop/_includes/components/indicator/tags.html

2: I copied this lines https://github.com/christoph-LSN/MT_Site/blob/b085bd12c5145e480f3c9108a60998548cfd4e99/_includes/components/indicator/tags.html#L15-18 an inserted them in lines 19-22 and changes 
`...reporting-status-label.html`
into
`data_status.html`

3: I created data_status.html in my _includes/components/indicator folder: https://github.com/christoph-LSN/MT_Site/blob/develop/_includes/components/indicator/data_status.html

4: I added a field `data_status` in my _prose.yml https://github.com/christoph-LSN/IM_Data/blob/36880e02385ca59199d3f28128b301296898797b/_prose.yml#L95

5: I added in my metadata file for indicator 1.1.1 `data_status: 2020` https://github.com/christoph-LSN/IM_Data/edit/develop/meta/1-1-1.md#L29

If I inspect the site https://christoph-lsn.github.io/MT_Site/1/ I see the following:

![image](https://user-images.githubusercontent.com/94443931/173057109-c9a77aad-b57b-43df-a8fc-df8d17fc4177.png)

It seems there is a new field called data_status but it's empty....

Do I miss something?

**Comment by LucyGwilliamAdmin** [comment, marked as answer]
@christoph-LSN what's the difference between what you want and the existing tags functionality?

**Reply by MoBosse** [reply]
@christoph-LSN if you just add 

> tags:
>   - Datenstnd: 2020

as described [here](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#tags), you will get the following result: 
![grafik](https://user-images.githubusercontent.com/50657629/173100894-5a62f69b-73be-4da2-91a9-56923be43d6d.png)
isn't this what you want (except for the typo)?

**Reply by LucyGwilliamAdmin** [reply]
@MoBosse thanks - that's what I was going to suggest 👍

**Comment by christoph-LSN** [comment]
@MoBosse Thank you! It works great!

![image](https://user-images.githubusercontent.com/94443931/173304521-aeff9cd5-9824-4f66-8878-5f216e5a47a6.png)
